pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
```

Comment:
Is setuptools strictly required if this is a pure Python package?

Reply:
Do you have a better suggestion? This works fine

Reply:
It was just a thought. I think hatchling seems to be the preference on pure Python packages these days. I think removing `setup.py` is probably sufficient to prevent common disasters

Reply:
I have had a couple of issues with hatchling in the past. setuptools hasn't ever really let me down.